pkg/crypto/bls/api.go

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package bls
+
+// Bls defines BBS+ signature scheme (https://eprint.iacr.org/2016/663.pdf, section 4.3).
+type Bls interface {
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Renamed to `BBS`